tests/validation/tests/v3_api/test_airgap.py

```diff
@@ -0,0 +1,455 @@
+import base64
+import os
+import pytest
+import time
+from lib.aws import AWS_USER
+from .common import (
+    AmazonWebServices, run_command, TEST_IMAGE, TEST_IMAGE_NGINX,
+    TEST_IMAGE_OS_BASE
+)
+from .test_custom_host_reg import (
+    random_test_name, RANCHER_SERVER_VERSION, HOST_NAME, AGENT_REG_CMD
+)
+from .test_create_ha import (
+    RANCHER_HA_CERT_OPTION, RANCHER_VALID_TLS_CERT, RANCHER_VALID_TLS_KEY
+)
+
+PRIVATE_REGISTRY_USERNAME = os.environ.get("RANCHER_BASTION_USERNAME")
+PRIVATE_REGISTRY_PASSWORD = os.environ.get("RANCHER_BASTION_PASSWORD")
+PROVIDER_ID = os.environ.get("RANCHER_PROVIDER_ID", "")
+NUMBER_OF_INSTANCES = int(os.environ.get("RANCHER_AIRGAP_INSTANCE_COUNT", "1"))
+IMAGE_LIST = os.environ.get("RANCHER_IMAGE_LIST", ",".join(
+    [TEST_IMAGE, TEST_IMAGE_NGINX, TEST_IMAGE_OS_BASE])).split(",")
+
+RANCHER_AG_INTERNAL_HOSTNAME = HOST_NAME + "-internal.qa.rancher.space"
```

Comment:
Can you add the deletion of the route53 record for `RANCHER_AG_INTERNAL_HOSTNAME ` here: 
https://github.com/rancher/rancher/blob/0d7c892c00fdd62e0d0673ef98c98bd802bb7a4e/tests/validation/tests/v3_api/common.py#L2648-L2651